ui/src/app/views/unregistered/component.spec.ts

```diff
@@ -85,6 +108,8 @@ describe('UnregisteredComponent', () => {
     router.navigateByUrl('/unregistered');
     tick();
     fixture.detectChanges();
+
+    unregisteredComponent = de.children[1].componentInstance;
```

Comment:
Could you add a comment on what element this is trying to grab? (Just confused by the array selection)

Is it the component as a whole?

Reply:
Added comment and used a slightly less opaque method instead.